backend/api/v1/endpoints/auth.py

```
@@ -24,59 +22,67 @@
 # Router
 router = APIRouter()
 
-# Signup Route
+# Dependency to get DB session
+def get_db():
```

Ruff (0.8.2)

Missing return type annotation for public function `get_db`

(ANN201)